### DISCUSSION TITLE
BUG: Payment customization (move operation) not working in Shopify Plus store with Shopify Payments/PayPal

### DISCUSSION THREAD

**Discussion by Oteiza-a** [Shopify/function-examples#497] [Feedback: Payment Customization API]
I am applying a Payment Customization move operation in a Shopify Plus store to switch the position of Shopify Payments credit card and PayPal, but even though the operations are correct, the outcome is not being reflected in the checkout.

Function input payment methods:
<img width="567" alt="Screenshot 2024-05-06 at 16 59 57" src="https://github.com/Shopify/function-examples/assets/49501058/ac36e0cd-c858-49b9-a42c-6ef96825491c">

Function output move operations:
<img width="872" alt="Screenshot 2024-05-06 at 17 00 41" src="https://github.com/Shopify/function-examples/assets/49501058/ae0b520c-b17a-4f22-a259-e6ba2d5599e5">

Result in the checkout page:
<img width="757" alt="Screenshot 2024-05-06 at 17 02 35" src="https://github.com/Shopify/function-examples/assets/49501058/67f287a8-239b-4f5d-924d-42e7d269b1b1">

The checkout of the merchant is a 3-page checkout, could this be causing some problem or incompatibility?

It's important to note too that hide operations work seamlessly in this same store.

**Comment by kylebuildsstuff** [comment]
Just ran into this issue also, more details here: https://github.com/Shopify/ui-extensions/issues/2235

**Comment by shivendra8004** [comment]
I am also facing same issue, Did you guys found any solution
